traducao do primeiro capitulo (#1)
jonasagx/infdesc: book/getting-started/getting-started.tex

\begin{proposition}
\label{propDivisibilityIsTransitive}
Seja $a,b,c \in \mathbb{Z}$. Se $c$ divide $b$ e $b$ divide $a$, então $c$ divide $a$.
\end{proposition}

\begin{cproof}

Suponha que $c$ divide $b$ e $b$ divide $a$. Por \Cref{defDivisionPreliminary}, segue que
\[ b=qc \quad \text{e} \quad a=rb \]
para alguns inteiros $q$ e $r$. Usando a primeira equação, podemos substituir $qc$ por $b$ na segunda equação, para obter
\[ a=r(qc) \]
Mas $r(qc) = (rq)c$, e $rq$ é um número inteiro, então segue de \Cref{defDivisionPreliminary} que $c$ divide $a$.
\end{cproof}

% To do: draw attention to the wording of the proof, and the 'unpack definition - do something - apply definition' style of the proof.

\begin{exercise}
\label{exDivisibilityIsLinear}
Sejam $a,b,d \in \mathbb{Z}$. Suponha que $d$ divide $a$ e $d$ divide $b$. Dados inteiros $u$ e $v$, prove que $d$ divide $au+bv$.
\end{exercise}

Alguns conceitos familiares, como par e impar, podem ser caracterizados em termos de divisibilidade.

\begin{definition}
\label{defEvenOdd}
\index{Inteiro!par}
\index{Inteiro!´impar}
Um inteiro $n$ é \textbf{par} se é divisível por $2$; de outra forma, $n$ é \textbf{impar}.
\end{definition}

> Não é apenas interessante saber quando um inteiro \textit{divide} outro; entretanto, provar que um inteiro \textit{não} divide outro é muito mais difícil. Na verdade, para provar que um inteiro $b$ não divide um inteiro $a$, devemos provar que $a \ne qb$ para \textit{qualquer} inteiro $q$. Veremos métodos para fazer isso em \Cref{chLogicalStructure}; esses métodos usam o seguinte resultado extremamente importante, que será a base de toda a \Cref{chNumberTheory}.

Author: ```suggestion
Não é apenas interessante saber quando um inteiro \textit{divide} outro; entretanto, provar que um inteiro \textit{não} divide outro é muito mais difícil. Na verdade, para provar que um inteiro $b$ não divide um inteiro $a$, devemos provar que $a \ne qb$ para \textit{qualquer} inteiro $q$. Veremos métodos para fazer isso em \Cref{chLogicalStructure}; esses métodos usam o seguinte resultado extremamente importante, que será a base de todo o \Cref{chNumberTheory}.
```

\begin{theorem}[Teorema da divisão, para ser repetido in \Cref{thmDivisionTheorem}]
\label{thmDivisionPreliminary}
\index{teorema da divisão}
Seja $a,b \in \mathbb{Z}$ com $b \ne 0$. Existe exatamente uma maneira de escrever
\[ a = qb + r \]
de tal modo que $q$ e $r$ são inteiros, e $0 \le r < b$ (se $b > 0$) ou $0 \le r < -b$ (se $b < 0$).
\end{theorem}

O número $q$ em \Cref{thmDivisionPreliminary} é chamado o \textbf{quociente}\index{quociente} de $a$ quando dividido por $b$, e o número $r$ é chamado o \textbf{restante}\index{restante}.

\begin{example}
O número $12$ deixa um  restante de $2$ quando dividido por $5$, desde que $12 = 2 \cdot 5 + 2$.
\end{example}

Aqui está um exemplo um pouco mais complicado.

\begin{proposition}
Suponha um número inteiro $a$ deixa um resto de $r$ quando dividido por um número inteiro $b$, e que $r>0$. Então $-a$ deixa um resto de $b-r$ quando dividido por $b$.
\end{proposition}

\begin{cproof}
Suponha que $a$ deixa um resto de $r$ quando dividido por $b$. Então
\[ a=qb+r \]

> Para algum número inteiro $q$. Um pouco de rendimento de álgebra

Author: ```suggestion
Para algum número inteiro $q$. Um pouco de álgebra nos dá
```

\[ -a = -qb-r = -qb-r+(b-b) = -(q+1)b + (b-r) \]

> Since $0<r<b$, Nós temos $0<b-r<b$. Por isso $-(q+1)$ é o quociente de $-a$ quando dividido por $b$, e $b-r$ é o restante.

Author: ```suggestion
Com $0<r<b$, nós temos $0<b-r<b$. Por isso $-(q+1)$ é o quociente de $-a$ quando dividido por $b$, e $b-r$ é o restante.
```

\end{cproof}

\begin{exercise}

> Prove que se um inteiro $a$ deixa um resto de $r$ quando dividido por um inteiro $b$, então $a$ deixa um resto de $r$ quando dividido por $-b$.

Author: ```suggestion
Prove que se um inteiro $a$ deixa um resto $r$ quando dividido por um inteiro $b$, então $a$ deixa um resto de $r$ quando dividido por $-b$.
```

\end{exercise}

Terminaremos esta parte sobre divisão de inteiros conectando-a com o material em bases numéricas --- podemos usar o teorema da divisão (\Cref{thmDivisionPreliminary}) para encontrar a expansão de base $b$ de um determinado número natural. É baseado na seguinte observação: o número natural $n$ cuja expansão de base $b$ é $d_rd_{r-1} \cdots d_1 d_0$ é igual a
\[ d_0 + b(d_1 + b(d_2 + \cdots + b(d_{r-1} + bd_r) \cdots)) \]
Além disso, $0 \le d_i < b$ para todo $i$. Em particular, $n$ deixa um resto de $d_0$ quando dividido por $b$. Por isso
\[ \frac{n-d_0}{b} = d_1 + d_2b + \cdots + d_rb^{r-1} \]

> A base -b $b$ expansão de $\frac{n-d_0}{b}$ é portanto

Author: ```suggestion
A expansão de base $b$ de $\frac{n-d_0}{b}$ é portanto
```

\[ d_rd_{r-1} \cdots d_1 \]

> Em outras palavras, o resto de $n$ quando dividido por $b$ é o último dígito de base $b$ de $n$, e então subtrair esse número de $n$ e dividir o resultado por $b$ trunca o final dígito. Repetir esse processo nos dá $d_1$, e depois $d_2$, e assim por diante, até chegarmos a $0$.

Author: ```suggestion
Em outras palavras, o resto de $n$ quando dividido por $b$ é o último dígito de base $b$ de $n$, e então subtrair esse número de $n$ e dividir o resultado por $b$ trunca o dígito final. Repetir esse processo nos dá $d_1$, e depois $d_2$, e assim por diante, até chegarmos a $0$.
```

Isso sugere o seguinte algoritmo para calcular a expansão de base $b$ de um número $n$:
\begin{itemize}

> \item \textbf{Passo 1.} SeJA $d_0$ seja o resto quando $n$ é dividido por $b$, e Seja $n_0=\frac{n-d_0}{b}$ seja o quociente. Consertar $i=0$.

Author: ```suggestion
\item \textbf{Passo 1.} Seja $d_0$ o resto quando $n$ é dividido por $b$, e seja $n_0=\frac{n-d_0}{b}$ seja o quociente. Fixe $i=0$.
```

> \item \textbf{Step 2.} Suponha $n_i$ e $d_i$ foram definidos. Se $n_i=0$, em seguida, prossiga para o Passo 3. Por outro lado define $d_{i+1}$ o seja o resto quando $n_i$ é dividido por $b$ e defina $n_{i+1} = \frac{n_i-d_{i+1}}{b}$. Aumente $i$ e repita a Etapa 2.

Author: ```suggestion
\item \textbf{Step 2.} Suponha $n_i$ e $d_i$ foram definidos. Se $n_i=0$, em seguida, prossiga para o Passo 3. Senão, defina $d_{i+1}$ como resto quando $n_i$ é dividido por $b$ e defina $n_{i+1} = \frac{n_i-d_{i+1}}{b}$. Incremente $i$ e repita a Etapa 2.
```

\item \textbf{Etapa 3.} A expansão base-$b$ de $n$ é
\[ d_id_{i-1} \cdots d_0 \]
\end{itemize}

\begin{example}
Calculamos a expansão de base $17$ de $15213$, usando as letras $\mathrm{A}$--$\mathrm{G}$ para representar os números $10$ até $16$.
\begin{itemize}
\item $15213 = 894 \cdot 17 + 15$, então $d_0=15=\mathrm{F}$ e $n_0=894$.
\item $894 = 52 \cdot 17 + 10$, então $d_1=10 = \mathrm{A}$ e $n_1=52$.
\item $52 = 3 \cdot 17 + 1$, então $d_2 = 1$ and $n_2=3$.
\item $3 = 0 \cdot 17 + 3$, então $d_3 = 3$ e $n_3=0$.

> \item A base -$17$ expansão de $15213$ é portanto $31\mathrm{AF}$.

Author: ```suggestion
\item A expansão base -$17$ de $15213$ é portanto $31\mathrm{AF}$.
```

}
\end{tikzpicture}
\end{center}
Os números racionais são aqueles da forma $\frac{a}{b}$, onde $a,b \in \mathbb{Z}$ e $b \ne 0$. Escrevemos $\mathbb{Q}$ \inlatex{mathbb\{Q\}}\lindexmmc{mathbb}{$\mathbb{A}, \mathbb{B}, \dots$} para o conjunto de todos os números racionais; portanto, a notação `$q \in \mathbb{Q}$' significa que $q$ é um número racional.\end{definition}

Os números racionais são um exemplo muito importante de um tipo de estrutura algébrica conhecida como \textit{campo} --- eles são particularmente centrais para a teoria algébrica dos números e para a geometria algébrica.

\subsection*{Números reais ($\mathbb{R}$)}

> Quantidade e mudança podem ser medidas abstratamente usando \textit{números reais}.

Author: ```suggestion
Quantidade e mudança podem ser medidas de forma abstrata usando \textit{números reais}.
```

\begin{definition}
\label{defRealsInformal}
Os \textbf{números reais} são os pontos na reta numérica. Escrevemos $\mathbb{R}$ \inlatex{mathbb\{R\}}\lindexmmc{mathbb}{$\mathbb{A}, \mathbb{B}, \dots$} para o conjunto de todos os números reais; portanto, a notação `$a \in \mathbb{R}$' significa que $a$ é um número real.

\end{definition}

> Os números reais são centrais para a análise real, um ramo da matemática introduzido em \Cref{chRealNumbers}. Eles transformam os racionais em um \textit{continuum} ao `preencher as lacunas'---especificamente, eles têm a propriedade de \textit{completude}, o que significa que se uma quantidade pode ser aproximada com precisão arbitrária por números reais, então essa quantidade é em si um número real.

Author: ```suggestion
Os números reais são centrais para a análise real, um ramo da matemática introduzido em \Cref{chRealNumbers}. Eles transformam os racionais em um \textit{continuo} ao `preencher as lacunas'---especificamente, eles têm a propriedade de \textit{completude}, o que significa que se uma quantidade pode ser aproximada com precisão arbitrária por números reais, então essa quantidade é em si um número real.
```

Podemos definir as operações aritméticas básicas (adição, subtração, multiplicação e divisão) sobre os números reais, e uma noção de ordenação dos números reais, em termos da reta numérica infinita.
\begin{itemize}
\item \textbf{Ordenando.} Um número real $a$ é menor que um número real $b$, escrito $a<b$, se $a$ estiver à esquerda de $b$ na reta numérica. As convenções usuais para os símbolos $\le$ \inlatex{le}\lindexmmc{le}{$\le$}, $>$ e $\ge$ \inlatex{ge}\lindexmmc{ge}{$\ge$ } se aplicam, por exemplo `$a \le b$' significa que $a < b$ ou $a = b$.

> \item \textbf{Adição.} Suponha que queiramos adicionar um número real $a$ a um número real $b$. Para fazer isso, \textit{traduzimos} $a$ em $b$ unidades para a direita --- se $b<0$ então isso equivale a traduzir $a$ em um número equivalente de unidades para a esquerda. Concretamente, faça duas cópias da reta numérica, uma acima da outra, com a mesma escolha de comprimento unitário; mova o $0$ da reta numérica inferior abaixo do ponto $a$ da reta numérica superior. Então $a+b$ é o ponto na reta numérica superior situado acima do ponto $b$ da reta numérica inferior.

Author: ```suggestion
\item \textbf{Adição.} Suponha que queremos adicionar um número real $a$ a um número real $b$. Para fazer isso, \textit{traduzimos} $a$ em $b$ unidades para a direita --- se $b<0$ então isso equivale a traduzir $a$ em um número equivalente de unidades para a esquerda. Concretamente, faça duas cópias da reta numérica, uma acima da outra, com a mesma escolha de comprimento unitário; mova o $0$ da reta numérica inferior abaixo do ponto $a$ da reta numérica superior. Então $a+b$ é o ponto na reta numérica superior situado acima do ponto $b$ da reta numérica inferior.

Aqui temos uma ilustração do fato que $(-3) + 5 = 2$:
```

\end{tikzpicture}}
\end{center}

> \item \textbf{Multiplicação.} Este é divertido. Suponha que queiramos multiplicar um número real $a$ por um número real $b$. Para fazer isso, \textit{escala} a reta numérica e talvez a \textit{reflete}. Concretamente, faça duas cópias da reta numérica, uma acima da outra; alinhe os pontos $0$ em ambas as retas numéricas e estique a reta numérica inferior uniformemente até que o ponto $1$ na reta numérica inferior esteja abaixo do ponto $a$ na reta numérica superior --- observe que se $a<0$ então a reta numérica deve ser refletida para que isso aconteça. Então $a \cdot b$ é o ponto na reta numérica superior situado acima de $b$ na reta numérica inferior.

Author: @GGG710 essa traducao esta ruim. Eu gostaria que voce explicasse o que foi feito sem usar os cognatos do ingles, que nesse caso nao traduzem bem a ideia

\end{itemize}

\begin{exercise}
Interprete as operações de subtração e divisão como transformações geométricas da reta numérica real.\end{exercise}

> Tomaremos como certas as propriedades aritméticas dos números reais neste capítulo, esperando até \Cref{secInequalitiesMeans} para afundar nossos dentes nos detalhes. Por exemplo, tomaremos como certas as propriedades básicas dos números racionais, por exemplo

Author: ```suggestion
Tomaremos como certas as propriedades aritméticas dos números reais neste capítulo, esperando até \Cref{secInequalitiesMeans} para nos aprofundamos nos detalhes. Por exemplo, tomaremos como certas as propriedades básicas dos números racionais, por exemplo
```

Author: @GGG710 essa traducao ao pe da letra ficou ruim porque se fala dessa forma em PT

\[ \frac{a}{b}+\frac{c}{d} = \frac{ad+bc}{bd} \quad \text{and} \quad \frac{a}{b} \cdot \frac{c}{d} = \frac{ac}{bd} \]

\subsection*{Números Racionais e Irracionais}
\label{pGettingStartedRationalNumbers}

Antes de podermos falar sobre números irracionais, devemos dizer o que são.

\begin{definition}
\label{defIrrationalNumber}
\index{numéro irracional}

> An \textbf{Número irracional}é um número que não é racional.

Author: ```suggestion
Um \textbf{Número irracional} é um número que não é racional.
```

\end{definition}

Ao contrário de $\mathbb{N},\mathbb{Z},\mathbb{Q},\mathbb{R},\mathbb{C}$, não existe uma única letra padrão que expresse os números irracionais. No entanto, até ao final de \Cref{secSetOperations}, seremos capazes de escrever o conjunto de números irracionais como `$\mathbb{R} \setminus \mathbb{Q}$'.

> Provar que um número real é \textit{irracional} não é particularmente fácil, em geral. Colocaremos o pé na porta permitindo-nos assumir o seguinte resultado, que é reafirmado e provado em \Cref{propSqrt2Irrational}.

Author: ```suggestion
Provar que um número real é \textit{irracional} não é particularmente fácil, em geral. Por hora vamos assumir o seguinte resultado, que é reafirmado e provado em \Cref{propSqrt2Irrational}.
```

\begin{proposition}
\label{propSqrt2IrrationalPreliminary}
O número real $\sqrt{2}$ é irracional. \qed
\end{proposition}

% To do: include proof, note that we will prove that fractions can be cancelled to lowest terms later.

Podemos usar o fato de que $\sqrt{2}$ é irracional para provar alguns fatos sobre a relação entre números racionais e números irracionais.

\begin{proposition}
Sejam $a$ e $b$ números irracionais. É possível que $ab$ seja racional.
\end{proposition}

\begin{cproof}
Seja $a=b=\sqrt{2}$. Então $a$ e $b$ são irracionais,e $ab=2=\frac{2}{1}$, o que é racional.
\end{cproof}

\begin{exercise}
Seja $r$ um número racional e $a$ um número irracional. Prove que é possível que $ra$ seja racional, e é possível que $ra$ seja irracional.\end{exercise}

% To do: set-builder notation and intervals

\subsection*{Números Complexos ($\mathbb{C}$)}

> Vimos que a multiplicação por números reais corresponde ao escalonamento e reflexão da reta numérica – escalonamento sozinho quando o multiplicando é positivo, e escalonamento com reflexão quando é negativo. Poderíamos alternativamente interpretar esta reflexão como uma \textit{rotação} de meia volta, uma vez que o efeito na reta numérica é o mesmo. Você pode então se perguntar o que acontece se girarmos em ângulos arbitrários, em vez de apenas meias voltas.

Author: @GGG710 Aqui voce traduziu `escale` (verbo) como escalonamento, o que e melhor que escalar. Por favor nao use google tradutor, e pior ainda nao use google tradutor sem revisar.

\draw (-3pt,-1) -- (3pt,-1);
\end{tikzpicture}
}
\caption{Ilustração do plano complexo, com alguns pontos rotulados.}
\label{figComplexNumbers}
\end{figure}

Existe um número complexo particularmente importante, $i$, que é o ponto no plano complexo exatamente uma unidade acima de $0$ --- isso é ilustrado em \Cref{figComplexNumbers}. A multiplicação por $i$ tem o efeito de girar o plano um quarto de volta no sentido anti-horário. Em particular, temos $i^2 = i \cdot i = -1$; os números complexos têm a surpreendente propriedade de que existem raízes quadradas de \textit{todos} os números complexos (incluindo todos os números reais).

Na verdade, todo número complexo pode ser escrito na forma $a+bi$, onde $a,b \in \mathbb{R}$; este número corresponde ao ponto no plano complexo obtido movendo $a$ unidades para a direita e $b$ unidades para cima, invertendo as direções como de costume se $a$ ou $b$ for negativo. A aritmética nos números complexos funciona da mesma forma que nos números reais; em particular, usando o fato de que $i^2=-1$, obtemos

> \[ (a+bi)+(c+di) = (a+c)+(b+d)i \quad \text{e} \quad (a+bi) \cdot (c+di) = (ac -bd) + (anúncio+bc)i \]

Author: ```suggestion
\[ (a+bi)+(c+di) = (a+c)+(b+d)i \quad \text{and} \quad (a+bi) \cdot (c+di) = (ac-bd) + (ad+bc)i \]
```

Discutiremos números complexos mais detalhadamente na parte deste capítulo sobre polinômios abaixo.

\subsection*{Polinômios}
\label{pGettingStartedPolynomials}

Os números naturais, inteiros, números racionais, números reais e números complexos são todos exemplos de \textit{semirings}, o que significa que eles vêm equipados com noções de adição e multiplicação bem comportadas.

\begin{definition}
\label{defPolynomialPreliminary}
\index{polinômios}
Seja $\mathbb{S} = \mathbb{N}$, $\mathbb{Z}$, $\mathbb{Q}$, $\mathbb{R}$ ou $\mathbb{C}$. Um (\textbf{univariado}) \textbf{polinômio sobre $\mathbb{S}$} no \textbf{indeterminado} $x$ é uma expressão da forma
\[ a_0 + a_1x + \cdots + a_nx^n \]
onde $n \in \mathbb{N}$ e cada $a_k \in \mathbb{S}$. Os números $a_k$ são chamados de \textbf{coeficientes} do polinômio. Se nem todos os coeficientes forem zero, o maior valor de $k$ para o qual $a_k \ne 0$ é chamado de \textbf{grau} do polinômio. Por convenção, o grau do polinômio $0$ é $-\infty$.
\end{definition}

> Polinômios de grau $1$, $2$, $3$, $4$ e $5$ são respectivamente chamados \textit{lineares}, \textit{quadratic}, \textit{cubic}, \textit{quartic} and \textit{quintic} polynomials.

Author: @GGG710 Por favor corrija os nomes dos polinomios

\begin{example}
As seguintes expressões são todas polinômios:
\[ 3 \qquad 2x-1 \qquad (3+i)x^2-x \]

> Seus diplomas são $0$, $1$ e $2$, respectivamente. Os dois primeiros são polinômios sobre $\mathbb{Z}$, e o terceiro é um polinômio sobre $\mathbb{C}$.

Author: ```suggestion
Seus graus são $0$, $1$ e $2$, respectivamente. Os dois primeiros são polinômios sobre $\mathbb{Z}$, e o terceiro é um polinômio sobre $\mathbb{C}$.
```

\end{notation}

Note que, se $A$ é qualquer um dos conjuntos $\mathbb{N}$, $\mathbb{Z}$, $\mathbb{Q}$, $\mathbb{R}$ ou $\mathbb{C}$, e $p(x)$ é um polinômio sobre $A$, então $p(\alpha) \in A$ para toda $\alpha \in A$.

\begin{example}
Seja $p(x)=x^3-3x^2+3x-1$. então $p(x)$ é um polinômio sobre $\mathbb{Z}$ com indeterminado $x$. Para qualquer número inteiro $\alpha$, o valor $p(\alpha)$ também será um número inteiro. Por exemplo
\[ p(0) = 0^3-3 \cdot 0^2 + 3 \cdot 0 - 1 = -1 \quad \text{and} \quad p(3) = 3^3 - 3 \cdot 3^2 + 3 \cdot 3 - 1 = 8 \]
\end{example}

\begin{definition}
\label{defRootOfPolynomial}
\index{raiz}
Seja $p(x)$ m polinômio. Uma \textbf{raiz} de $p(x)$ é um número complexo $\alpha$ de tal modo que $p(\alpha)=0$.
\end{definition}

> A \textit{fórmula quadrática} (\Cref{thmQuadraticFormula}) nos diz que as raízes do polinômio $x^2+ax+b$, onde $a,b \in \mathbb{C}$, são precisamente o complexo números

Author: ```suggestion
A \textit{fórmula quadrática} (\Cref{thmQuadraticFormula}) nos diz que as raízes do polinômio $x^2+ax+b$, onde $a,b \in \mathbb{C}$, são precisamente os números complexos
```

\[ \frac{-a+\sqrt{a^2-4b}}{2} \quad \text{e} \quad \frac{-a-\sqrt{a^2-4b}}{2} \]

> Observe que evitamos o símbolo `$\pm$', que é comumente encontrado em discussões sobre polinômios quadráticos. O símbolo `$\pm$' é perigoso porque pode suprimir a palavra `e' ou a palavra `ou', dependendo do contexto --- este tipo de ambigüidade não é algo com o qual desejaremos lidar ao discutir a lógica estrutura de uma proposição em \Cref{chLogicalStructure}!

Author: ```suggestion
Observe que evitamos o símbolo `$\pm$', que é comumente encontrado em discussões sobre polinômios quadráticos. O símbolo `$\pm$' é perigoso porque pode suprimir a palavra `e' ou a palavra `ou', dependendo do contexto --- este tipo de ambigüidade não é algo com o qual desejaremos lidar ao discutir a estrutura lógica de uma proposição em \Cref{chLogicalStructure}!
```
